Add Cell Line Prediction Analysis - Complete Figure 3

gwaygenomics opened the pull request:
Adding a notebook that applies the Ras Classifier Predictions to two Cell Line Datasets

gwaygenomics commented:
@dhimmel - particularly interested in your expertise with data usage/licensing and data visualization in this PR

dhimmel commented on environment.yml:
```diff
@@ -6,11 +6,12 @@ dependencies:
 - anaconda::jupyter=1.0.0
 - anaconda::numpy=1.11.2
-- anaconda::pandas=0.19.2
+- anaconda::pandas=0.20.2
```
20.3 is available now

gwaygenomics commented:
PR ready for review again @dhimmel - I updated one of the cell line figures to include just one dimension.

I also added some minor changes to the ras figure script, since this is included in updates to figure 3
